bundles/org.openhab.binding.warmup/README.md

```diff
@@ -76,6 +100,19 @@ These run mode statuses are defined for the API. The descriptions are based on i
 | relay      | Relay            | Unknown                                                                         |
 | previous   | Previous         | Unknown                                                                         |
 
+## Rule Actions
+
+### setOverride(temperature, duration)
+
+Sets a temporary temperature override on the device
+
+ Parameters:
+
+| Name        | Type                      | Description                                                             |
+|-------------|---------------------------|-------------------------------------------------------------------------|
+| temperature | QuantityType<Temperature> | Override temperature. Must be between 5°C and 30°C                      |
```

Comment:
I think this should be Number:Temperature

Reply:
I disagree - this is a rule/thing action so the parameter is a java type rather than an item type. There's not many examples in the repo, I guess most have stuck with primitives - but I cribbed this from the Astro binding https://github.com/openhab/openhab-addons/blob/main/bundles/org.openhab.binding.astro/README.md. Seems there's not a lot of consistency as to how actions are presented unlike things & channels. Does this make sense?

Reply:
Quick check, i might be wrong, but isn't the Astro binding different? As in, it returns a quantitype and does not take a quantitytype parameter?

Reply:
Agree the `QuantityType` is the return rather than the argument but I'm not sure that makes a difference? We're operating in java here, there's no item to hold an item type. Another example is [ecobee](https://www.openhab.org/addons/bindings/ecobee/#thing-actions), the docs are formatted a bit differently but it has a number of methods which accept `QuantityType`s

Reply:
There's also https://www.openhab.org/addons/bindings/energidataservice/#calculatecheapestperiod-from-duration-and-power with some examples for both DSL and JavaScript: https://www.openhab.org/addons/bindings/energidataservice/#thing-actions-example

Reply:
I've tested this in DSL - I'm a bit of a dinosaur and not got into JS rules yet, have we reached the point where JS rules are the default minimum and DSL is nice to have?

With hindsight I think the code I used to create `WarmupActions` class had different method signatures for DSL vs. JS, but IIRC the method DSL calls is just a wrapper to the method JS calls directly, so I ticked that off as covered.

I will have a crack and see if I can a JS rule working soon - thanks for the reference @jlaur, should get me up and running quickly.

Reply:
Also useful to add that (simplified) to the readme as example.

Reply:
@clinique @lsiepel Readme updated with JS and DSL examples